packages/trpc/server/routers/loggedInViewer/deleteCredential.handler.ts

```diff
@@ -27,12 +27,12 @@ type DeleteCredentialOptions = {
 };
 
 export const deleteCredentialHandler = async ({ ctx, input }: DeleteCredentialOptions) => {
-  const { id, externalId } = input;
+  const { id, externalId, teamId } = input;
 
   const credential = await prisma.credential.findFirst({
     where: {
       id: id,
-      userId: ctx.user.id,
+      OR: [{ userId: ctx.user.id }, { teamId }],
```

Comment:
Don't use org here, if either it's provided just use that for the filter.
```suggestion
const whereDelete: Prisma.CredentialWhereUniqueInput  = {
   id,
};
if (teamId) {
whereDelete.teamId = teamId;
} else {
whereDelete.userId = ctx.user.id;
}

...
where: whereDelete,
...

```